### ISSUE TITLE
when "Teleporter Buttons = true", log is spammed endlessly unless both teleporters are purchased and out of storage at the same time

### ISSUE THREAD

**Issue by notandvm** [d-rafferty/lethal-tube-removal#28]
pretty much exactly as title

when the custom button positions are enabled, if both teleporters are not unlocked and out of storage, the log gets flooded with null references to either the teleporter or inverse teleporter button, depending on which is not unlocked/out of storage

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
LethalTubeRemoval.Patches.TubeRemovalPatch.<TeleporterButtons>g__TeleporterStuff|4_0 () (at <4880d9d1af7d4889a83e38d0cb8ff6e6>:0)
(wrapper dynamic-method) ShipTeleporter.DMD<ShipTeleporter::Update>(ShipTeleporter)
```

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
LethalTubeRemoval.Patches.TubeRemovalPatch.<InverseTeleporterButtons>g__InverseTeleporterStuff|5_0 () (at <4880d9d1af7d4889a83e38d0cb8ff6e6>:0)
(wrapper dynamic-method) ShipTeleporter.DMD<ShipTeleporter::Update>(ShipTeleporter)
```

**Comment by d-rafferty** [comment]
At one point there was a check to ensure this didn't happen, must have accidentally had it on a branch that got deleted. I'll get this taken care of.

**Comment by d-rafferty** [comment]
Should be resolved in v1.6.4 which will be live shortly. Fixed with #30
